yucca/run/run_finetune.py

```diff
@@ -54,119 +51,84 @@ def main():
-            folder=[join(yucca.__path__[0], "training", "trainers")],
-            class_name=trainer_name,
-            current_module="yucca.training.trainers",
-        )
-        checkpoint = join(
-            yucca_models, source_task, model, dimensions, trainer_name + "__" + plans, str(folds), "checkpoint_final.model"
-        )
-        trainer = trainer(
-            model=model,
-            model_dimensions=dimensions,
-            task=task,
-            folds=folds,
-            plan_id=plans,
-            starting_lr=lr,
-            loss_fn=loss,
-            momentum=momentum,
-            continue_training=True,
-            checkpoint=checkpoint,
-            finetune=True,
-            fast_training=fast_training,
-        )
-
-        command_used = "yucca_train " + " ".join(f"-{k} {v}" for k, v in vars(args).items())
-        trainer.set_train_command(command_used)
-
-        trainer.run_training()
-    if ensemble:
-        print("Starting ensemble training. Model dimensions will automatically be set to 2D.")
-        dimensions = "2D"
-        views = ["X", "Y", "Z"]
-        for view in views:
-            trainer = recursive_find_python_class(
-                folder=[join(yucca.__path__[0], "training", "trainers")],
-                class_name=trainer_name,
-                current_module="yucca.training.trainers",
-            )
-            plan_and_view = plans + view
-            checkpoint = join(
-                yucca_models,
-                source_task,
-                model,
-                dimensions,
-                trainer_name + "__" + plan_and_view,
-                str(folds),
-                "checkpoint_final.model",
-            )
-            trainer = trainer(
-                model=model,
-                model_dimensions=dimensions,
-                task=task,
-                folds=folds,
-                plan_id=plan_and_view,
-                starting_lr=lr,
-                loss_fn=loss,
-                momentum=momentum,
-                continue_training=True,
-                checkpoint=checkpoint,
-                finetune=True,
-                fast_training=fast_training,
-            )
-
-            command_used = "yucca_train " + " ".join(f"-{k} {v}" for k, v in vars(args).items())
-            trainer.set_train_command(command_used)
-
-            trainer.run_training()
+    manager = recursive_find_python_class(
+        folder=[join(yucca.__path__[0], "training", "trainers")],
+        class_name=manager_name,
+        current_module="yucca.training.trainers",
+    )
+    # checkpoint = join(
+    #    yucca_models,
+    #    source_task,
+    #    model + "__" + dimensions,
+    #    manager_name + "__" + planner,
+    #    f"fold_{str(folds)}",
+    #    f"version_{str(version)}",
+    #    "checkpoints",
+    #    f"{checkpoint}.ckpt",
+    # )
+
+    manager = manager(
+        ckpt_path=checkpoint,
+        continue_from_most_recent=False,
+        deep_supervision=False,
```

Comment:
I guess we can just delete `deep_supervision` here since the default value is already False